cadence/contracts/bridge/FlowEVMBridge.cdc

```diff
@@ -453,7 +690,78 @@ contract FlowEVMBridge : IFlowEVMNFTBridge {
         // Associate the deployed contract with the given type & return the deployed address
         let erc721Address = decodedResult[0] as! EVM.EVMAddress
         FlowEVMBridgeConfig.associateType(forNFTType, with: erc721Address)
-        return erc721Address
+        return EVMOnboardingValues(
+            evmContractAddress: erc721Address,
+            name: name,
+            symbol: symbol,
+            decimals: nil
+        )
+    }
+
+    /// Deploys templated ERC721 contract supporting EVM-native asset bridging to Cadence
+    ///
+    /// @param forTokenType: The Cadence Type of the FungibleToken.Vault
+    ///
+    /// @returns The EVMAddress of the deployed contract
+    ///
+    access(self)
+    fun deployERC20(_ forTokenType: Type): EVMOnboardingValues {
+        // Retrieve the Cadence type's defining contract name, address, & its identifier
+        var name = FlowEVMBridgeUtils.getContractName(fromType: forTokenType)
+            ?? panic("Could not contract name from type: ".concat(forTokenType.identifier))
+        let identifier = forTokenType.identifier
+        let cadenceAddress = FlowEVMBridgeUtils.getContractAddress(fromType: forTokenType)
+            ?? panic("Could not derive contract address for token type: ".concat(identifier))
+        // Assign a default symbol
+        var symbol = "BRDG"
```

Comment:
Rather than having a default, I think it might be better to panic if neither metadata field is filled in as expected.

I don't think anyone would want to accidentally have their token bridged with a name/ticker that can't be changed on all evm consuming indexed websites which they didnt have control over

Reply:
Looking at MetadataViews, there isn't a view that supports symbol. So we can make this call for fungible tokens, but we'll have to consider how to dynamically create a symbol if a project doesn't implement the `EVMBridgedMetadata` view. Maybe the first `n` characters of the name with special characters and spaces removed? Have any thoughts?